src/test/java/com/faforever/server/integration/legacy/transformer/UserDetailsResponseTransformerTest.java

```diff
@@ -127,4 +134,24 @@ public void transformHandleCountryNull() throws Exception {
     Map<String, Object> me = (Map<String, Object>) result.get("me");
     assertThat(me.get("country"), is(""));
   }
+
+  @Test
+  @SuppressWarnings("unchecked")
+  public void transformHandleTimeZoneNull() throws Exception {
+    Map<String, Serializable> result = UserDetailsResponseTransformer.INSTANCE.transform(new UserDetailsResponse(
+      1,
+      TEST_USERNAME,
+      "CH",
+      null,
+      new UserDetailsResponse.Player(
+        new Rating(1200d, 200d),
+        new Rating(900d, 100d),
+        12,
+        null
+      )
+    ));
+
+    Map<String, Object> me = (Map<String, Object>) result.get("me");
+    assertThat(me.get("time_zone"), is(""));
+  }
```

Comment:
This test could be integrated into the transformHandleCountryNull() test, but timezone may be null even when the country is not null.